examples/meta/src/binary/svmocas.sg

```diff
@@ -0,0 +1,23 @@
+File f_feats_train = csv_file("../../data/fm_train_real.dat")
+File f_feats_test = csv_file("../../data/fm_test_real.dat")
+File f_labels_train = csv_file("../../data/label_train_twoclass.dat")
+
+#![create_features]
+Features feats_train = features(f_feats_train)
+Features feats_test = features(f_feats_test)
+Labels labels_train = labels(f_labels_train)
+#![create_features]
+
+#![create_classifier]
+Machine svm = machine("SVMOcas", features=feats_train, labels=labels_train, use_bias=False, C1=0.9, C2=0.9, epsilon=0.00001)
+#![create_classifier]
+
+#![train_and_apply]
+svm.train()
+Labels labels_predict = svm.apply(feats_test)
```

Comment:
could you extract the label vector in both examples, then this will also be added to the integration test, which is useful